xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -175,6 +175,7 @@ class CAddonRecording : public PVR_RECORDING
   const std::string m_recordingId;
   const std::string m_title;
   const std::string m_episodeName;
+  int m_episodePartNumber;
```

Comment:
Please remove this line. A member is only needed for `const char*`API struct members.

Reply:
Sure, I copied the const char* API change logic for the original

Reply:
From what I see, it is still there? Please remove.